src/xai_grok/schemas/message_request.py

```diff
@@ -0,0 +1,23 @@
+import typing
+
+import pydantic
+
+from xai_grok.schemas.message import Message
+from xai_grok.schemas.message_metadata import MessageMetadata
+from xai_grok.schemas.message_text_content import MessageTextContent
```

Comment:
The `MessageToolChoice` is imported but never used in the code. This creates an unnecessary dependency that should be removed.